src/App.tsx

```diff
@@ -27,6 +28,30 @@ function App() {
                 >
                     Learn React
                 </a>
+                <Router>
+                    <ul>
+                        <li><Link to="/complextree"> Comlex Tree </Link></li>
+                        <li><Link to="/simple"> Simple Tree </Link></li>
+                    </ul>
+                    <Routes>
+                        <Route path="/complextree"
+                               element={
+                                    <TopLevelComponent
+                                    firstParagraph="This is the first paragraph"
+                                    secondParagraph="This is the second paragraph"
+                                    imageLink={nasalogo}
+                                    />}
+                        />
+                        <Route path="/simple"
+                               element={
+                                   <NasaData
+                                       firstParagraph="This is the first paragraph"
```

Comment:
let's say I don't want to have a paragraph but a simple div. take a look at 
https://felixgerschau.com/react-component-composition/